migrations/2024-12-18-200602_optimize_get_random_community/up.sql

```diff
@@ -0,0 +1,20 @@
+-- * `smallint` range from https://www.postgresql.org/docs/17/datatype-numeric.html
+-- * built-in `random` function has `VOLATILE` and `PARALLEL RESTRICTED` according to:
+--   * https://www.postgresql.org/docs/current/parallel-safety.html#PARALLEL-LABELING
+--   * https://www.postgresql.org/docs/17/xfunc-volatility.html
+CREATE FUNCTION random_smallint ()
+    RETURNS smallint
+    LANGUAGE sql
+    VOLATILE PARALLEL RESTRICTED RETURN
+    -- https://stackoverflow.com/questions/1400505/generate-a-random-number-in-the-range-1-10/1400752#1400752
+    -- (`+ 1` makes the upper bound inclusive)
+    trunc ((random() * (32768 + 32767 + 1)) - 32768
```

Comment:
Could probably just type out `65536` , I'm sure it doesn't really make a difference tho. The link being there is what's most important.

Reply:
changed it to 65536